src/NHibernate.Test/ExceptionsTest/SerializationFixture.cs

```diff
@@ -0,0 +1,24 @@
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using NUnit.Framework;
+
+namespace NHibernate.Test.ExceptionsTest
+{
+	[TestFixture]
+	public class SerializationFixture
+	{
+		[Test]
+		public void InstantiationExceptionSerialization()
```

Comment:
The pattern used for exception serialization with type was not tested. So adding this test.